src/main/scala/org/hammerlab/guacamole/assembly/DeBruijnGraph.scala

```diff
@@ -293,9 +304,9 @@ object DeBruijnGraph {
     graph
   }
 
-  def mergeKmers(kmers: Seq[Kmer]): Sequence = {
-    val head = kmers.headOption.map(h => h.take(h.length - 1)).getOrElse(Seq.empty)
-    val rest = kmers.map(_.last)
+  def mergeKmers(kmers: Seq[Kmer], kmerSize: Int): Sequence = {
+    val head = kmers.headOption.getOrElse(Seq.empty)
+    val rest = kmers.tail.flatMap(kmer => kmer.takeRight(kmer.length - kmerSize + 1))
```

Comment:
so I guess you've adapted this to work on merged "k-mers"?

Calling merged-kmers (that are longer than k) "kmers" makes this a little confusing; can we try to distinguish the var names or types, or comment it at all?